tools/cloud-build/python-integration-tests/test.py

```diff
@@ -0,0 +1,428 @@
+# Copyright 2024 "Google LLC"
```

Comment:
`cloud-buld` looks like a wrong place, lets think about it/

Reply:
Moved outside to tools for now.